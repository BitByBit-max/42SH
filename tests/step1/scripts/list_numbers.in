echo pairs list:
seq 0 2 10
echo

echo impairs list:
seq 1 2 9
