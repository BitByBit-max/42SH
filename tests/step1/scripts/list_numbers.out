pairs list:
0
2
4
6
8
10

impairs list:
1
3
5
7
9
